src/Panel/TwigPanel.php

```diff
@@ -38,7 +43,18 @@ class TwigPanel extends DebugPanel
     public function data(): array
     {
         return [
-            'templates' => TreeScanner::all(),
+            'templates' => TreeScanner::all(static::$extensions),
```

Comment:
Not entirely related to this specific change, but loading all the templates here seems like something that could get unwieldy in a large application. Should we only list out the templates that are relevant or used in the request?

Reply:
I don't see the usefulness of this panel to be honest. I'd rather disable it.

Reply:
I agree, I wouldn't mind if the panel was dropped altogether for now.

Reply:
Or at least not loaded automatically by default (which `Plugin::bootstrap()` currently does).

Reply:
Opt-in sounds reasonable.

Reply:
Opened #33 for it.